src/consts.hpp

```diff
@@ -3,14 +3,14 @@
 #include <limits>
 #include <madrona/math.hpp>
 #include <madrona/types.hpp>
- 
+
 namespace gpudrive {
 
 namespace consts {
 
-inline constexpr madrona::CountT kMaxAgentCount = 128;
+inline constexpr madrona::CountT kMaxAgentCount = 32;
 inline constexpr madrona::CountT kMaxRoadEntityCount = 6000;
-inline constexpr madrona::CountT kMaxAgentMapObservationsCount = 200;
+inline constexpr madrona::CountT kMaxAgentMapObservationsCount = 64;
```

Comment:
This seems like it might need some consideration. I think the agent count is probably okay but are you sure 64 map points is enough?